test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -706,55 +719,91 @@ def __are_requirements_matching(param_dic=None, test=None):
             param_count['num_' + key] = len(param_dic[key])
         for pk in param_count:
             if param_count[pk] < eff_tc_req[pk]:
-                return False
+                _msg = 'available ' + pk + " ("
+                _msg += str(param_count[pk]) + ") is less than required " + pk
+                _msg += " (" + str(eff_tc_req[pk]) + ")"
+                logger.error(_msg)
+                return _msg
         for hostname in param_dic['moms']:
             si = SystemInfo()
             si.get_system_info(hostname)
             available_sys_ram = getattr(si, 'system_ram', None)
             if available_sys_ram is None:
-                return False
+                _msg = 'failed to get ram info on host: ' + hostname
+                logger.error(_msg)
+                return _msg
             elif eff_tc_req['min_mom_ram'] >= available_sys_ram:
-                return False
+                _msg = 'available ram is less than test requirement on host: '
```

Comment:
I just noticed that you haven't changed this log similar to the log on line 722.
we should log the available and required values - 
available ram (<value>) is less than the minimum required (<value>) for test exexcution